api/inventory/v1/agents.proto

```diff
@@ -1181,6 +1224,54 @@ message ChangeQANMongoDBProfilerAgentParams {
   common.MetricsResolutions metrics_resolutions = 4;
 }
 
+// Add/Change QANMongoDBMongologAgent
+
+message AddQANMongoDBMongologAgentParams {
+  // The pmm-agent identifier which runs this instance.
+  string pmm_agent_id = 1 [(validate.rules).string.min_len = 1];
```

Comment:
```suggestion
  string pmm_agent_id = 1 [(validate.rules).string.uuid = true];
```

Reply:
I know, we have to refactor in many places. I thought this is new code, so we can go ahead with new validation.

Reply:
Ok, makes sense. I will change that.

Reply:
Ah, yes! Ok, revoking the suggestion.